fitbenchmarking/fitbenchmarking
# 386 add levmar

#### Description of Work

Fixes #386 


#### Testing Instructions

1. Choose levmar in the options
2. Check that the results are consistent
3.

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
